model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/AffectedEndpointInfo.kt

```diff
@@ -0,0 +1,18 @@
+package org.digma.intellij.plugin.model.rest.insights
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import java.beans.ConstructorProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class AffectedEndpointInfo
```

Comment:
1. Why do we need the new type `AffectedEndpointInfo` , what happend to `EndpointInfo`?
2. does the backened support it already? because if not we need to migrate it carefully.

Reply:
1) Its different schemas. And related only to `SpanScalingRootCauseInsight`
<img width="369" alt="image" src="https://user-images.githubusercontent.com/8687862/233271446-a32e4f43-2dcb-4f9c-b909-64ecebd66bd6.png">
2) Yes

Reply:
I see there's an inheritance in the backend, lets do the same on the plugin (client) side.
we already have `SpanInfo` on the client side as well

Reply:
Done